src/main/java/teammates/ui/controller/InstructorSearchPageData.java

```diff
@@ -21,10 +41,71 @@
     public boolean isSearchCommentForStudents;
     public boolean isSearchCommentForResponses;
     public boolean isSearchForStudents;
+    private List<SearchCommentsForStudentsTable> searchCommentsForStudentsTables;
+    private List<SearchCommentsForResponsesTable> searchCommentsForResponsesTables;
+    private List<SearchStudentsTable> searchStudentsTables;
     
     public InstructorSearchPageData(AccountAttributes account) {
         super(account);
     }
+    
+    public void init(CommentSearchResultBundle commentSearchResultBundle, 
+                         FeedbackResponseCommentSearchResultBundle feedbackResponseCommentSearchResultBundle,
+                             StudentSearchResultBundle studentSearchResultBundle, String searchKey, int totalResultsSize,
+                                 boolean isSearchCommentForStudents, boolean isSearchCommentForResponses, boolean isSearchForStudents) {
+        this.commentSearchResultBundle = commentSearchResultBundle;
+        this.feedbackResponseCommentSearchResultBundle = feedbackResponseCommentSearchResultBundle;
+        this.studentSearchResultBundle = studentSearchResultBundle;
+        this.searchKey = searchKey;
+        this.totalResultsSize = totalResultsSize;
+        this.isSearchCommentForStudents = isSearchCommentForStudents;
+        this.isSearchCommentForResponses = isSearchCommentForResponses;
+        this.isSearchForStudents = isSearchForStudents;
+        
+        setSearchCommentsForStudentsTables();
+        setSearchCommentsForResponsesTables();
+        setSearchStudentsTables();
+    }
+    
+    public String getSearchKey() {
+        return sanitizeForHtml(searchKey);
+    }
+    
+    public boolean getIsSearchCommentForStudents() {
+        return isSearchCommentForStudents;
+    }
+    
+    public boolean getIsSearchCommentForResponses() {
+        return isSearchCommentForResponses;
+    }
+    
+    public boolean getIsSearchForStudents() {
```

Comment:
I would personally recommend using the isSomething() naming convention rather than getIsSomething() like here...